src/hotspot/share/opto/memnode.cpp

```diff
@@ -1098,6 +1098,17 @@ Node* LoadNode::can_see_arraycopy_value(Node* st, PhaseGVN* phase) const {
   return nullptr;
 }
 
+static Node* see_through_inline_type(PhaseValues* phase, const MemNode* load, Node* base, int offset) {
```

Comment:
Is there a specific reason that you made this `static` and not a member function of `MemNode` (then you can use `this` instead of `load`)?